### ISSUE TITLE
It it possible to do a multi-AZ deployment on AWS?

### ISSUE THREAD

**Issue by adun153** [cloudfoundry/cf-mysql-release#52]
I was wondering if it is at all possible.

**Comment by shalako** [comment]
It is possible, but you'd have to manually create a manifest to do it. 

We have an epic coming up very soon to update our templates to support a three-AZ deployment. Once this work is done, you'll be able to create a manifest for a multi-AZ deployment by running the generate_manifest script. 

You can track our progress toward "mysql multi-az aws" on our public Tracker project: https://www.pivotaltracker.com/n/projects/969486

**Comment by menicosia** [comment]
Hi there, @adun153,

Just updating this ticket to say that manual work is no longer required, as of cf-mysql-release [v20](https://github.com/cloudfoundry/cf-mysql-release/releases/tag/v20) we've released templates that support multi-AZ deployments by default.

I'll close this ticket now, please let us know if there's anything additional that you were hoping for that's not covered in this issue. Seeing as this is rather old, it's probably best to open a new issue based on your experience with v20, rather than re-opening this one.

--
Marco Nicosia
Product Manager
Pivotal Cloud Foundry
